api/src/main/java/net/starype/quiz/api/game/answer/MultipleChoiceCorrectness.java

```diff
@@ -0,0 +1,49 @@
+package net.starype.quiz.api.game.answer;
+
+import net.starype.quiz.api.game.util.MathUtils;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class MultipleChoiceCorrectness implements CorrectnessEvaluator {
+
+    private Set<String> acceptedAnswer;
+    private LossFunction lossFunction;
+    private double punitiveRatio = 1.0F;
+
+    public MultipleChoiceCorrectness(Set<Answer> acceptedAnswer,
+                                     LossFunction lossFunction,
+                                     double punitiveRatio)
+    {
+        this.acceptedAnswer = acceptedAnswer
+                .stream()
+                .map(s -> s.getAnswerText().toLowerCase())
```

Comment:
Is it really a good idea to have string processing here? It might be time to implement an AnswerProcessor to avoid those kinds of random processing at random places in the code